### ISSUE TITLE
"Material Files keeps stopping" again...

### ISSUE THREAD

**Issue by PrSunflower** [zhanghai/MaterialFiles#388]
Hi, the issue #370 seemed to be solved but unfortunately the same crash happened again. 😕

### I have a picture in my phone's internal storage

![Screenshot_20201122-193507](https://user-images.githubusercontent.com/59576952/99914226-36573e00-2cfc-11eb-8ea7-13e0f8ee6f55.jpg)

### Material Files takes ~2s to load the picture

![Screenshot_20201122-193518](https://user-images.githubusercontent.com/59576952/99914262-63a3ec00-2cfc-11eb-9f57-e754e9ca3235.png)

### But it never loads and crashes

![Screenshot_20201122-193535](https://user-images.githubusercontent.com/59576952/99914274-761e2580-2cfc-11eb-8776-da3356d8c33d.png)

<br>

<br>

## Here is the picture in question

[IMG_20201117_140217_1.zip](https://github.com/zhanghai/MaterialFiles/files/5580071/IMG_20201117_140217_1.zip)

**Comment by zhanghai** [comment]
There hasn't been an app release since #370 was fixed, so this is likely a duplicate of the same issue.

Regarding the release, actually I've already have 80% SMB support ready for a next major release (I could even play videos smoothly from an SMB share with my dev version), but lacking the traction to finish it recently, partly for personal reasons, partly due to the UI code I had to deal with. So I'm coming up with a plan that deprecates the MD1 design in a next minor release without the SMB feature, while shipping existing bug fixes with it. And the next major release would drop MD1 design, simplifying UI code for SMB and allowing more things for MD2 like AMOLED dark and theme colors.
